src/codegen.cpp

```diff
@@ -6432,6 +6432,7 @@ extern "C" void jl_init_codegen(void)
 #endif
 
 #ifdef USE_POLLY
+    cl::ParseEnvironmentOptions("Julia", "JULIA_POLLY_ARGS");
```

Comment:
My point was that having this mechanism for passing arguments to LLVM is fine, but calling it `JULIA_POLLY_ARGS` is not – because that's not what it does. I would second what @vchuravy suggested and just call this variable `JULIA_LLVM_ARGS` and not make it's parsing directly connected to Polly at all. If you then figure out a way to scope arguments to LLVM only to Polly (or whatever) that can be a separate configuration feature.